index.html

```diff
@@ -0,0 +1,715 @@
+                    </div>
+                </div>
+            </div>
+            <div class="deal-adapt">
+                <div class="deal-holder deal-4"></div>
+                <div class="deals-info-4 deal-push">
+                    <div class="deals-title"><strong>The Ultimate Artificial Intelligence Scientist Certification
+                            Bundle</strong></div>
+                    <div class="deals-source">
+                        <div class="price"><s>$799</s></div> $34.99
+                    </div>
+                </div>
+            </div>
+            <div class="deal-adapt">
+                <div class="deal-holder deal-5"></div>
+                <div class="deals-info-5 deal-push">
+                    <div class="deals-title"><strong>JumpStory™️ Authentic Stock Photography: Lifetime
+                            Membership</strong></div>
+                    <div class="deals-source">
+                        <div class="price"><s>$2340</s></div> $99.99
+                    </div>
+                </div>
+            </div>
+            <div class="deal-adapt">
+                <div class="deal-holder deal-6"></div>
+                <div class="deals-info-6 deal-push">
+                    <div class="deals-title"><strong>The Speed Reading & Memory Mastery Bundle</strong></div>
+                    <div class="deals-source">
+                        <div class="price"><s>$995</s></div> $29.99
+                    </div>
+                </div>
+            </div>
+            <div class="deal-adapt">
+                <div class="deal-holder deal-7"></div>
+                <div class="deals-info-7 deal-push">
+                    <div class="deals-title"><strong>The Trainz Railroad Simulator Platinum Edition Bundle</strong>
+                    </div>
+                    <div class="deals-source">
+                        <div class="price"><s>$279</s></div> $39.99
+                    </div>
+                </div>
+            </div>
+            <div class="deal-adapt">
+                <div class="deal-holder deal-8"></div>
+                <div class="deals-info-8 deal-push">
+                    <div class="deals-title"><strong>Stackskills Unlimited: Lifetime Access</strong></div>
+                    <div class="deals-source">
+                        <div class="price"><s>$1495</s></div> $59
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+    <footer>
+        <div class="footer-1">
+            <div class="icons">
+                <i class="fab fa-facebook-square"></i>
+                <i class="fab fa-twitter"></i>
+                <i class="fab fa-youtube"></i>
+                <i class="fab fa-instagram"></i>
+                <i class="fab fa-pinterest-p"></i>
+                <i class="fas fa-wifi"></i>
+                <i class="fas fa-envelope"></i>
+            </div>
+            <div class="footer-list">
```

Comment:
- [ ] After the last change, the layout is breaking in the mobile screen size.


| Yours | Original |
|:---:|:---:|
| ![image](https://user-images.githubusercontent.com/61479648/89461080-b94e8b80-d741-11ea-966c-7fbb69d2bf1a.png)| ![image](https://user-images.githubusercontent.com/61479648/89461128-cc615b80-d741-11ea-9945-dce0faf6565e.png) |